### PR TITLE
Added flores200 spm model

### PR DESCRIPTION
**Pull request by mjpost**
Supplements #168.

Documentation forthcoming.

### CODE REVIEW

File: sacrebleu/tokenizers/tokenizer_spm.py [hunk 1]
```diff
@@ -11,11 +11,29 @@
 sacrelogger = logging.getLogger('sacrebleu')
 
 
+SPM_MODELS = {
+    "spm": {
+        "url": "https://dl.fbaipublicfiles.com/fairseq/models/flores/sacrebleu_tokenizer_spm.model",
+        "signature": "spm-flores",
+    },
+    # same as the default of "spm"
```
Comment (martinpopel) [inline sacrebleu/tokenizers/tokenizer_spm.py:19]: Why is the same model available under two names and two different signatures?

Comment (mjpost) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: The newer name is more descriptive, the older one for backwards compatibility. Maybe not a good idea?

Comment (martinpopel) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: Do we need backward compatibility here? Maybe we can deprecate "spm" - switch to "flores101" (including the signature), but issue a warning that "spm" will be removed in the next version of sacrebleu.

Comment (mjpost) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: It's listed in § 5.2 in the [flores 101 paper](https://aclanthology.org/2022.tacl-1.30.pdf), so I'd hate to remove it entirely. But in hindsight it's definitely named too generically.

Comment (martinpopel) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: I see there just a sentence "SpBLEU is integrated into sacrebleu for ease of use as the spm tokenizer". I don't see `-tok spm`. If you think so, we don't need to deprecate `-tok spm`, we can just keep it as an alias for `-tok flores101`. My main point is that the signature should be clear (and I think two signatures for the same thing may be confusing).

Comment (mjpost) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: image

I'll give it some thought, and clean it up somehow. I agree it's simpler to have one name, and maybe we can just print a deprecation warning.

Comment (mjpost) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: This is what I plan to do (retain `spm` for backward compat). Sound good?

Comment (mjpost) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: So to be clear: You can use `-tok spm|flores101|flores200`, which will generate signatures of "flores101", "flores101", and "flores200", respectively.

Comment (mjpost) [reply sacrebleu/tokenizers/tokenizer_spm.py:19]: There is also a note printed if you use `-tok spm`, that the signature has changed.

**Review by martinpopel — APPROVED** [review]

**Pull request merged**